command/sub8_missions/missions/arm_torpedos.py

```diff
@@ -27,8 +32,8 @@ class FireTorpedos(object):
     Its goal is to search for a target on the torpedo board and fire at it.
     '''
     TIMEOUT_SECONDS = 30
-    Z_PATTERN_RADIUS = 0.3
-    Y_PATTERN_RADIUS = 2.0
+    Z_PATTERN_RADIUS = 0.01
```

Comment:
Looks like torpedo board is 1.2 - 1.8m above the floor according to the rules... So might be a good idea to go that depth

Reply:
And 0.01 seems pretty small for searching in z